docs/make.jl

```diff
@@ -3,7 +3,12 @@ using Documenter, Conductor
 makedocs(sitename="Conductor.jl",
          doctest = false,
          modules = [Conductor],
```

Comment:
https://github.com/SciML/DiffEqFlux.jl/pull/721/files#diff-4aae2d1c783cade58bd2cb13748da956e568b7f2aed5fafd9e2a46fb97daf613R14-R21 could be helpful

Reply:
Doc tests are enabled and strict errors are enabled for all checks. All examples are setup to use `jldoctest` or, in some cases, `@example` paired with hidden `@assert` lines.